server/apps/immich/src/api-v1/asset/asset.service.ts

```diff
@@ -432,6 +432,10 @@ export class AssetService {
         result.push({ id, status: DeleteAssetStatusEnum.SUCCESS });
         deleteQueue.push(asset.originalPath, asset.webpPath, asset.resizePath);
 
+        if (asset.encodedVideoPath) {
```

Comment:
You can actually add this on line 433, it is fine if it is undefined. You will probably need to update a test or two though.

Reply:
Thanks, that makes things less cluttered. About the tests: I have little experience with that, even less in this scope. I guess this is the purpose of the `asset.service.spec.ts` file. I hope I understood correctly what I need to do here. Otherwise, a little info on what to roughly add is highly appreciated 😅.